src/Analysis/Ast/Impl/Indexing/ISymbolIndex.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.Python.Parsing.Ast;
+
+namespace Microsoft.Python.Analysis.Indexing {
+    internal interface ISymbolIndex {
```

Comment:
What happened to the query to get a document's HierarchicalSymbol?

Reply:
I added it to the interface. And updated the tests to use the interface